src/snap/metric.cc

```diff
@@ -199,6 +217,10 @@ int Metric::get_int_data() const {
   return rpc_metric_ptr->int32_data();
 }
 
+long int Metric::get_int64_data() const {
```

Comment:
Does `int64_data()` throw? It's generally good to mark functions that don't throw as `noexcept` for performance reasons. It is an API contract though, so if you're not sure, better not add it.